build.sbt

```diff
@@ -256,7 +256,8 @@ lazy val commonSettings =
 val flinkV = "1.16.0"
 val avroV = "1.11.0"
 //we should use max(version used by confluent, version acceptable by flink), https://docs.confluent.io/platform/current/installation/versions-interoperability.html - confluent version reference
-val kafkaV = "3.2.3"
+val kafkaClientV = "3.2.3"
+val kafkaServerV = "3.3.1"
```

Comment:
There is a dependency on `kafka-clients` in kafka server, so I don't think it's a good idea to mix versions...

Reply:
Yes, I noticed this when the PR failed.